internal/flag/flag.go

```diff
@@ -529,9 +529,16 @@ func JSONOutput() Bool {
 	}
 }
 
-func ProcessGroup() String {
+func ProcessGroup(desc string) String {
+
+	if desc == "" {
+		desc = "The target process group"
+	}
+
 	return String{
 		Name:        flagnames.ProcessGroup,
-		Description: "The target process group",
+		Description: desc,
+		Shorthand:   "g",
+		Aliases:     []string{"process-group"},
```

Comment:
Could we agree on using `--process-group` as main flag name, `--group` as alias and `-g` as shorthand? 

The reasoning is that in docs and forum topics we refer to this feature as _Process Groups_, calling them groups don't help the brain muscle.

Reply:
Done!